go/oasis-test-runner/oasis/oasis.go

```diff
@@ -144,6 +181,31 @@ func (n *Node) LoadIdentity() (*identity.Identity, error) {
 	return identity.Load(n.dir.String(), factory)
 }
 
+// Start starts the node.
+func (n *Node) Start() error {
+	args := newArgBuilder()
+	for _, f := range n.features {
+		if err := f.AddArgs(args); err != nil {
+			return fmt.Errorf("oasis/node: failed to add arguments for feature on node %s: %w", n.Name, err)
+		}
+		if cf, ok := f.(CustomStartFeature); ok {
+			if len(n.features) != 1 {
+				panic("oasis/node: trying to perform customized node start with multiple features attached")
```

Comment:
I assume a fixture can still configure this? It resulting in a panic is probably not ideal and it may be better if this returned an error instead?

Reply:
The idea here was that if a feature requires completely custom startup, it's likely to clash with everything else, since all the "regular" features probably also expect regular startup. In that sense, I don't think just returning an error makes sense. Alternatively, it could just let all features build up the args and then check if there's at most one custom feature - then panic if there's more, otherwise hand over startup to it at the end.

Reply:
So the problem I have with panicking is that this could be configured through an external fixture (so user configuration) and then would cause the test runner to just crash which seems non-ideal? It would be better if it exited with a nicer error instead. But not a critical thing, we can start with a panic and improve usability later.